src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmailController.php

```diff
@@ -223,6 +223,8 @@ public function deleteAction($mailId)
     /**
      * Processes test email sending.
      *
+     * @AdminSecurity("is_granted('create', request.get('_legacy_controller')) || is_granted('update', request.get('_legacy_controller')) || is_granted('read', request.get('_legacy_controller'))", message="Access denied.")
```

Comment:
After checking the code, I think the AdminSecurity was not used there because the response but must be JSON. I think `@AdminSecurity()` works for HTML response only.